src/server.ts

```diff
@@ -13,5 +13,5 @@ app.use('/api', routes);
 app.use('/docs', swaggerUi.serve, swaggerUi.setup(specs));
 
 app.listen(port, () => {
-  console.log(`Server is running on port ${port}`);
+  console.log(`Database Connection status: Success\nRunning Port: ${port}`);
```

Comment:
No, this is just the app starting, not the database. You should add other lines of code to authenticate to sequelize.
Here is [guide](https://sequelize.org/docs/v6/getting-started/#testing-the-connection)